src/app/move-popup/sockets.html

```diff
@@ -4,12 +4,22 @@
     <div class="item-socket"
       ng-class="{ disabled: !socketInfo.enabled }"
       ng-repeat="socketInfo in category.sockets track by $index">
-      <img class="item-mod"
-        ng-src="{{ socket.displayProperties.icon | bungieIcon }}"
-        ng-class="{ notChosen: socket != socketInfo.plug }"
-        ng-repeat="socket in socketInfo.plugOptions track by $index"
-        press-tip="{{ socket.displayProperties.description }}{{ socketInfo.enableFailReasons }}"
-        press-tip-title="{{ socket.displayProperties.name }}">
+      <div class="socket-container"
+           ng-repeat="socket in socketInfo.plugOptions track by $index">
+        <svg ng-if="socket != socketInfo.plug && socket.bestRated" width="34" height="34">
+          <circle
+            class="talent-node-best-rated-circle"
+            r="5"
+            cx="25"
+            cy="6"
+          />
+        </svg>
```

Comment:
I don't think we need to go so far as to use an SVG element here. We could also have a `div` with `border-radius: 50%`.